### DISCUSSION TITLE
[gatsby-source-wordpress]: custom createStaticFiles path

### DISCUSSION THREAD

**Discussion by yesh** [gatsbyjs/gatsby#32498] [Ideas / Feature Requests]
Hi,
I'm having an hard time with the gatsby-source-wordpress plugin parsing media/attachments urls because of the wp installation of my client.

### The case

The client uses a bedrock (https://roots.io/bedrock/) installation for wordpress, to enhance security and maintenance.

### The issue

Bedrock uses a different folder structure in which the uploads are in `app/uploads` folder path instead of the classic `wp-content/uploads`

### Feature request

I know it is a niche use case, but it would be good to create an option to change the source path for the static files generation and url replacement.

```
 options: {
    html: {
      createStaticFiles: true,
      sourcePath: `whatever/uploads`
    },
  },
```

hope it gets considered!

**Comment by justbitdev** [comment]
Same issue here, any workaround?

**Comment by steveariss** [comment]
I also have this issue. And it effects my ability to serve pdf's correctly :(
